src/main/java/uk/gov/digital/ho/hocs/document/JSONToSimpleTextConverter.java

```diff
@@ -55,6 +55,7 @@ private void buildString(JsonNode node, String keyName, int level) {
         } else {
             String textValue = node.textValue();
             if (keyName.equals("complaintText")) {
+                textValue = textValue.replaceAll("[\\n]", NEW_LINE_STR);
```

Comment:
NEW_LINE_STR is `\n         `, any reason for the weird amount of spaces?

Reply:
It's to match how the document should be formatted with indentation. See the screenshot I've attached to the ticket, easier to figure it out by viewing the behaviour

Reply:
Yep makes more sense 👍🏽